Sources/Testing/ExitTests/WaitFor.swift

```diff
@@ -218,7 +218,7 @@ func wait(for processHandle: HANDLE) async throws -> ExitCondition {
   }
 
   // FIXME: handle SEH/VEH uncaught exceptions.
-  return .exitCode(CInt(bitPattern: status))
+  return .exitCode(CInt(bitPattern: numericCast(status)))
```

Comment:
```suggestion
  return .exitCode(CInt(bitPattern: CUnsignedInt(status)))
```

As above, we don't otherwise use `numericCast`.